beacon_node/beacon_chain/src/migrate.rs

```diff
@@ -334,16 +334,33 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> BackgroundMigrator<E, Ho
         );
 
         for (head_hash, head_slot) in heads {
+            // Load head block. If it fails with a decode error, it's likely a reverted block,
+            // so delete it from the head tracker but leave it and its states in the database
+            // This is suboptimal as it wastes disk space, but it's difficult to fix. A re-sync
+            // can be used to reclaim the space, which in practice is likely to be miniscule.
```

Comment:
```suggestion
            // can be used to reclaim the space.
```

In practice if you've left the node running for a week past the fork boundary, your disk will be full :sweat_smile: